src/bootstrap/lib.rs

```diff
@@ -501,6 +501,9 @@ impl Build {
         if self.config.jemalloc {
             features.push_str("jemalloc");
         }
+        if self.config.llvm_enabled() {
+            features.push_str(" llvm");
```

Comment:
Maybe rename to `cg_llvm`?

Reply:
Perhaps? It's all internal anyway and can change at any time, I sort of prefer just `llvm` but feel free to send a PR after this if you feel differently.